sync-core/src/main/java/com/cloudant/sync/replication/Replication.java

```diff
@@ -41,6 +43,12 @@
 @Deprecated
 public abstract class Replication {
 
+    final List<HttpConnectionRequestFilter> requestFilters = new ArrayList
+            <HttpConnectionRequestFilter>();
+    final List<HttpConnectionResponseFilter> responseFilters = new ArrayList
+            <HttpConnectionResponseFilter>();
```

Comment:
I think it does too. However that being said I implemented it this way because the subclasses use public instance variables to interact with the source and target for the replication so I reasoned doing it this way is more in keeping with what we already had, which I think makes the classes less confusing